pkgs/development/python-modules/transformers/default.nix

```diff
@@ -18,13 +18,13 @@
 
 buildPythonPackage rec {
   pname = "transformers";
-  version = "4.0.0";
+  version = "4.0.1";
```

Comment:
```suggestion
  version = "4.0.1";
  disabled = isPy39;
```
```
    def test_with_optional(self):
>       parser = HfArgumentParser(OptionalExample)

tests/test_hf_argparser.py:139:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
src/transformers/hf_argparser.py:38: in __init__
    self._add_dataclass_arguments(dtype)
src/transformers/hf_argparser.py:71: in _add_dataclass_arguments
    elif hasattr(field.type, "__origin__") and issubclass(field.type.__origin__, List):
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = typing.List, cls = typing.Union

    def __subclasscheck__(self, cls):
        if isinstance(cls, _SpecialGenericAlias):
            return issubclass(cls.__origin__, self.__origin__)
        if not isinstance(cls, _GenericAlias):
>           return issubclass(cls, self.__origin__)
E           TypeError: issubclass() arg 1 must be a class

/nix/store/zpims2ycz1lp5rgvqskcx5ycwp7rxvx2-python3-3.9.0/lib/python3.9/typing.py:789: TypeError
```

Reply:
Thanks! I also removed the unused `boto3` argument.